src/AM.Condo/Targets/Publish/Go.targets

```
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <Target Name="GoPublish" Condition=" $(GoPublish) ">
+      <PropertyGroup>
+        <GoPublishOptions Condition=" '$(GoPublishOptions)' == '' ">$(GO_PUBLISH_OPTIONS)</GoPublishOptions>
+      </PropertyGroup>
+
+      <ItemGroup>
+        <GoPublishTargets Include="$(GO_PUBLISH_TARGETS)" />
+
+        <GoPublishArgs Include="@(GoPublishTargets)">
+          <OS>$([System.String]::new('%(Identity)').Split('/')[0].ToLower())</OS>
+          <Arch>$([System.String]::new('%(Identity)').Split('/')[1].ToLower())</Arch>
+          <Ext Condition=" '%(GoPublishArgs.OS)' == 'windows' ">.exe</Ext>
```

Can we use GoFileExtension here?

```suggestion
          <Ext>$(GoFileExtension)</Ext>
```

---

we could use it as a placeholder for `.exe`. we still need the condition since this is based on user input via `GO_PUBLISH_TARGETS`. so whenever the OS is windows in the variable array we append .exe to the Ext property. 

for instance `windows/amd64;windows/arm;dawrin/arm`
```
OS -> windows
Arch -> amd64
Ext -> .exe

OS -> windows
Arch -> arm
Ext -> .exe

OS -> dawrin
Arch -> amd64
Ext -> null
```

---

Ahh; got it.